DSharpPlus.Commands/Processors/SlashCommands/SlashCommandProcessor.cs

```diff
@@ -929,4 +930,48 @@ private void ValidateSlashCommand(Command command, IReadOnlyDictionary<string, s
             );
         }
     }
+
+    private async Task<IReadOnlyList<DiscordApplicationCommand>> RegisterGlobalCommandsRetryingAsync
+    (
+        IEnumerable<DiscordApplicationCommand> commands
+    )
+    {
+        while (true)
```

Comment:
i think we should still have some kind of max retries, maybe 5 or 10

Reply:
maybe, probably via config

Reply:
not sure if this has been fixed yet but I'd reiterate what plerx said, it's important to not keep retrying indefinitely. Especially if the issue ends up being an _us_ issue. We can not just assume our request is "correct". Especially since this is Discord we're talking about.